.github/workflows/integration-tests.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     - name: Setup Go
       uses: actions/setup-go@v5
       with:
-        go-version: "1.22"
+        go-version: "1.24"
```

Comment:
```suggestion
        go-version-file: go.mod
```
This would be a bit better, so that we don't have to update this again. But I don't insist on it.